src/less/actionToolbar.less

```diff
@@ -50,29 +45,9 @@
     }
 
     .dropdown-menu {
-      color: #fff;
-      background: @action-toolbar-dropdown-bg;
       border: 0;
       margin-top: 0;
       border-radius: 0;
-
-      li a,
-      li button {
-        transition: none;
-
-        &:hover {
-          background: @action-toolbar-dropdown-hover-bg;
-
-          &.disabled,
-          &:disabled {
-            background: none;
```

Comment:
We're missing all the disabled styles. Need to prevent a background from displaying on hover for disabled items.